addons/project/models/project.py

```diff
@@ -2803,14 +2803,14 @@ def _get_project_tags_domain(self, domain, project_id):
     @api.model
     def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=False, lazy=True):
         if 'project_id' in self.env.context:
-            tag_ids = self._name_search()
+            tag_ids = self._name_search(limit=0)
```

Comment:
Actually, I am wondering if it is not better to set it as `None` instead since I see the ORM prefers set the limit to `None` in all their other methods. 🤔 

```suggestion
            tag_ids = self._name_search(limit=None)
```